index.d.ts

```diff
@@ -118,6 +125,31 @@ export interface FastifyDynamicSwaggerOptions extends FastifySwaggerOptions {
    * Overwrite the route schema
    */
   transform?: Function;
+
+  refResolver?: {
+    /** Clone the input schema without changing it. Default to `false`. */
+    clone?: boolean;
+    buildLocalReference: (
+      /** The `json` that is being resolved. */
+      json: JSONValue,
```

Comment:
`'hello'`, `1`, `true` ... etc are valid according to this type.

Reply:
ahh i see. i'll add the JSONObject type as well :)